docs/upgrading.md

```diff
@@ -0,0 +1,19 @@
+# Hawtio Operator Upgrading Guide
+
+This document describes important changes between hawtio-operator versions and instructions on how to upgrade your operator installation.
+
+- [hawtio-operator 0.4.0](#hawtio-operator-0.4.0)
+
+## hawtio-operator 0.4.0
+
+### `spec.rbac.enabled` field is removed from CRD
+
+From 0.4.0 RBAC support is always enabled, so the field to toggle the feature is removed from the CRD ([#48](https://github.com/hawtio/hawtio-operator/pull/48)). If your `Hawtio` custom resource contains the following field (whether it is `true` or `false`), remove it from the yaml definition.
+
+```yaml
+spec:
+    rbac:
+        enabled: true
+```
+
+If you want to maintain the old `rbac.enabled=false` behaviour, you can still leave it without providing the `hawtio-rbac` ConfigMap which includes `ACL.yaml`, and RBAC will not be activated effectively.
```

Comment:
Is it really the case? My understanding is that RBAC will be activated, but the default ACL will be used.

Reply:
Thanks, good catch. I still think it's true that the old `rbac.enabled=false` behaviour is maintained when not providing the `hawtio-rbac` configmap, according to:
https://github.com/hawtio/hawtio-operator/commit/ae97905d7b8b530fb26aa54c84178e55409ff846
So, to be more accurate, with the older versions RBAC is in fact still activated even when you apply `rbac.enabled=false` but the default ACL is used. Is that correct?

Reply:
Hmm, looks like it's not quite right... `HAWTIO_ONLINE_RBAC_ACL=<empty>` and providing no `HAWTIO_ONLINE_RBAC_ACL` appears to make the difference. When an empty `HAWTIO_ONLINE_RBAC_ACL` env var is provided hawtio-online docker image still turns on RBAC with the default ACL.yaml as you said.

OK, I'll modify it accordingly again. Thanks for your feedback!